content/integrations/google_cloud_big_query.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Datadog-Google BigQuery Integration
+integration_title: Google BigQuery
+kind: integration
+doclevel: basic
+newhlevel: true
+git_integration_title: google_cloud_big_query
+---
+
+# Overview
+BigQuery is Google's fully managed, petabyte scale, low cost enterprise data warehouse for analytics.
+
+Get metrics from Google BigQuery to:
+
+* Visualize the performance of your BigQuery queries
+* Correlate the performance of your BigQuery queries with your applications
+
+# Installation
+
+If you haven't already, set up the [Google Cloud Platform integration first](/integrations/google_cloud_platform). There are no other installation steps that need to be performed.
```

Comment:
> There are no other installation steps that need to be performed.

I think we can either remove this sentence, or shorten it to something like "That's all!"

> Google Cloud Platform integration first

The 'first' is unnecessary since this is actually the only step.